middlewares/content-security-policy/README.md

```diff
@@ -1,6 +1,6 @@
 # Content Security Policy middleware
 
-Content Security Policy helps prevent unwanted content being injected into your webpages. This can mitigate cross-site scripting (XSS) vulnerabilities, malicious frames, unwanted trackers, and much more.
+Content Security Policy (CSP) helps prevent unwanted content from being injected/loaded into your webpages. This can mitigate cross-site scripting (XSS) vulnerabilities, Clickjacking, formajacking, malicious frames, unwanted trackers, client-side injected malware, and other [web client-side attacks](https://cspscanner.com/client-side-attacks).
```

Comment:
1. Could you make "clickjacking" lowercase?
1. Should "formajacking" be "formjacking"?
1. Could you remove "client-side injected malware"? It feels redundant with the last clause.
1. Could you remove the link?